api/v1alpha1/hivelocitymachinetemplate_types.go

```diff
@@ -45,6 +45,8 @@ type HivelocityMachineTemplateStatus struct {
 // +kubebuilder:resource:path=hivelocitymachinetemplates,scope=Namespaced,categories=cluster-api,shortName=capihvcmt
 // +kubebuilder:printcolumn:name="Image",type="string",JSONPath=".spec.template.spec.imageName",description="Image name"
 // +kubebuilder:printcolumn:name="Type",type="string",JSONPath=".spec.template.spec.type",description="Server type"
+// +kubebuilder:printcolumn:name="Reason",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].reason"
```

Comment:
do we have a conditions.SetSummary for the template as well? If not, we don't need this

Reply:
okay